Packs/CortexResponseAndRemediation/Playbooks/playbook-Uncommon_execution_of_ODBCConf.yml

```diff
@@ -0,0 +1,715 @@
+    task:
+      id: bcca051a-ab0b-4b59-8846-c6b7238fe153
+      version: -1
+      name: Is There a Host Risk Score?
+      description: This task checks if the host risk score was retrieved.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "14"
+      "yes":
+      - "13"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Core.RiskyHost.risk_level
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 940,
+          "y": 240
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "13":
+    id: "13"
+    taskid: 5170b127-ee82-4589-874a-6b668e00b2c5
+    type: condition
+    task:
+      id: 5170b127-ee82-4589-874a-6b668e00b2c5
+      version: -1
+      name: Is the Risk Score High
+      description: "this tasks checks if the host risk score is \"High\"."
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "14"
+      Malicious:
+      - "4"
+    separatecontext: false
+    conditions:
+    - label: Malicious
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: Core.AnalyticsPrevalence.Process.value
+            iscontext: true
+          right:
+            value:
+              simple: "False"
+          ignorecase: true
+        - operator: isEmpty
+          left:
+            value:
+              simple: alert.cgosigner
+            iscontext: true
+      - - operator: isEqualString
+          left:
+            value:
+              simple: Core.RiskyHost.risk_level
+            iscontext: true
+          right:
+            value:
+              simple: High
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 660,
+          "y": 420
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "14":
+    id: "14"
+    taskid: 7ce3bc2a-81a0-42e7-8d82-c6f35d296cbf
+    type: regular
+    task:
+      id: 7ce3bc2a-81a0-42e7-8d82-c6f35d296cbf
+      version: -1
+      name: Check For Specific Alerts
+      description: |-
+        Searches Demisto alerts. A summarized version of this scrips is available with the summarizedversion argument.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Administrator-Guide/Permission-Management
```

Comment:
This doesn't apply to XSIAM. No such field for Limited User Role, so we need to remove 551 to 553